pkgs/development/libraries/blitz/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, lib, fetchurl, pkg-config, gfortran, texinfo, python
+
+# Select SIMD alignment width (in bytes) for vectorization.
+, simdWidth ? 1
+
+# Pad arrays to simdWidth by default?
+# Note: Only useful if simdWidth > 1
+, enablePadding ? false
+
+# Activate serialization through Boost.Serialize?
+, enableSerialization ? true, boost ? null
+
+# Activate test-suite?
+# WARNING: Some of the tests require up to 1700MB of memory to compile.
+, doCheck ? true
+
+}:
+
+assert enableSerialization -> boost != null;
+
+let
+  inherit (lib) optional optionals;
+in
+
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
I have left some of this intact, since optional and optionals are both needed and boost is only required if serialization is enabled.